sdk/eventhub/event-processor-host/CONTRIBUTING.md

```diff
@@ -22,36 +22,8 @@ There are many ways that you can contribute to the Azure Event Processor Host pr
 * Submit a feature request
 
 ## Contributing Code
-To contribute code you need to issue a Pull Request against the develop branch. All code submissions will be reviewed and tested by the team, and those that meet a high bar for both quality and design/roadmap appropriateness will be merged into the source. Be sure to follow the existing file/folder structure when adding new boards or sensors.
-
-You must sign a [Contribution License Agreement](https://cla.microsoft.com/) ([CLA](https://cla.microsoft.com/)) before submitting a Pull Request. To complete the CLA, you will need to submit the request via the form and then electronically sign the CLA when you receive the email containing the link to the document.
-
-## Big contributions
-If your contribution is significantly big it is better to first check with the project developers in order to make sure the change aligns with the long term plans. This can be done simply by submitting a question via the GitHub Issues section.
-
-## Things to keep in mind when contributing
-Some guidance for when you make a contribution:
-
-* Add/update unit tests and code as required by your change
-* Make sure you run all the unit tests on the affected platform(s)/languages. If the change is in common code, generally running on one platform would be acceptable.
-* Run end-to-end tests or simple sample code to make sure the lib works in an end-to-end scenario.
-
-## Building the library
-- Install typescript, ts-node globally (optional, but very useful)
-```
-npm i -g typescript
-npm i -g ts-node
```

Comment:
So is the recommendation now to just have rush drive all the typescript installation?

Reply:
I don't expect npm would work well for our track 2 libraries when we have dependencies to versions net yet released.

Reply:
Less so that as if you look in the main CONTRIBUTING.md we don't tell the user how to get started with TypeScript either

Reply:
I could add the following part to the main file.  It could be helpful to have global install of typescript and ts-node.  For installing dependencies and building for our libraries I think `rush` is still the way to go.

> - Install typescript, ts-node globally (optional, but very useful)	
> ```	
> npm i -g typescript	
> npm i -g ts-node
> ```

Reply:
Actually this file should be removed actually.  We did similar change for event hubs and service bus  in 73f7f90c4 but missed this one.